### PR TITLE
Hws 179 git hub 138 show of scope data is confusing

### PR DESCRIPTION
**Pull request by YoniEisipsSNN**
Enhancement which allows for better formatting when printing the ScopeData object. Standardize output from get_data()

### CODE REVIEW

File: src/oscilloscope/types.jl [hunk 1]
```diff
@@ -86,17 +86,23 @@ struct ScopeData
 end
 
 function Base.show(io::IO, ::MIME"text/plain", data_array::AbstractArray{ScopeData})
-    for idx in 1:length(data_array)
+    for idx in eachindex(data_array)
```
Comment (mofii) [inline src/oscilloscope/types.jl:89]: nice

**Review by mofii — COMMENTED** [review]
This looks pretty good to me. Anything you'd still want to add since it's a draft?

**Comment by YoniEisipsSNN** [comment]
> This looks pretty good to me. Anything you'd still want to add since it's a draft?

Mostly I want you to have access to be able to test, suggest a change, or discuss a feature. If you feel this is a good solution to get_data() inconsistency then it can be marked no longer draft and merged! I can’t think of anything else we’d need to include.

**Review by mofii — APPROVED** [review]
👍

**Pull request merged**